Casks/y/yes24-ebook.rb

```diff
@@ -9,9 +9,8 @@
 
   livecheck do
     url "https://cremaupdate.k-epub.com/sv_update.aspx?usrid=&old=0"
-    strategy :xml do |xml|
-      xml.elements["//VERSION"]&.text&.strip
-    end
+    # Since the version field may be null, use the path field instead.
+    regex(%r{/v?(\d+(?:\.\d+)+)/YES24eBook\.dmg}i)
```

Comment:
```suggestion
    regex(%r{/v?(\d+(?:\.\d+)+)/YES24eBook\.dmg}i)
    strategy :xml do |xml, regex|
      url = xml.elements["//PATH"]&.text&.strip
      match = url.match(regex) if url

      match[1]
    end
```

We should continue using `strategy :xml` and working with the parsed XML content.